src/translation/translation_nl_NL.ts

```diff
@@ -171,7 +171,7 @@
     <message>
         <location filename="../aboutdlgbase.ui" line="59"/>
         <source>TextLabelVersion</source>
-        <translation>TextLabelVersion</translation>
+        <translation>Tekstlabel versie</translation>
```

Comment:
Not sure. I doubt it.

Reply:
@henkdegroot Can you take a look at this, please?

Reply:
I was checking some strings on Weblate and came across this untranslated text, I assumed it was a new text and therefore add dutch translation for it.
If it should not be translatable, then it should be marked as such to avoid confusion like this. Specially when using a tool like weblate.

Reply:
OK, so:
1. We need to confirm whether or not it _should_ be translated
2. We need to mark the string somehow that will be visible and understood by translators as not to be translated (ideally hiding it)

I can take a look at the first point but I've no idea on the second.

Reply:
From looking at the code, it makes no difference:
```
src/util.cpp:742:    lblVersion->setText ( GetVersionAndNameStr() );
...
src/util.cpp:1570:    strVersionText += QCoreApplication::tr ( "%1, Version %2", "%1 is app name, %2 is version number" ).arg ( APP_NAME ).arg ( VERSION );
```
So whatever's there gets overwritten with a translated value.  Maybe changing the placeholder to "XXX, MMMM 88.88.88" would do?

Reply:
I believe in the ui files, you can just indicate if the text used in the ui file should be translatable or not. If not translatable, then I believe lupdate will not extract the text from the ui file for translation. It should still allow you to "set/overwrite" the value in the cpp files and string in these files will still be extract by lupdate for translation.

It will basically change:

          <string>TextLabelVersion</string>

into

          <string notr="true">TextLabelVersion</string>

in the ui file.

To be honest, I did not test this...but would make sense to me it works like this.
If this is the case, we might want to update a few more ui files and mark text as not translatable to minimize translation effort. What do you think?

Reply:
> If this is the case, we might want to update a few more ui files and mark text as not translatable to minimize translation effort. What do you think?

Probably worth a separate issue for 3.10.0 but yes.

For now, this can be left as is, as it's not used.